### DISCUSSION TITLE
Red ball demo without impedance

### DISCUSSION THREAD

**Discussion by kt10aan** [robotology/community#44] [Q&A] [answered]
Hello,

I think that, the red ball demo had two versions: with impedance and without.

But now, only the impedance version seems to be around. However, the impedance mode in `iCubOsaka01` is not working properly right now (to be investigated). If I switch the `impedance` to `off` in the config files of the module, in icus-basic-demos, would it produce the non-impedance enabled behaviour of the demo?

**Comment by pattacini** [comment, marked as answer]
Hi @kt10aan,

The so-called `red ball demo` has still the two modes available. 
And yes, if you switch off that flag you'll get the `impedance` disabled.

**Comment by kt10aan** [comment]
@pattacini thanks!

**Comment by pattacini** [comment]
Closing...
For any other doubt, please comment on https://github.com/robotology/icub-basic-demos/issues.
